frontend/client/modules/create/utils.ts

```diff
@@ -127,13 +127,15 @@ export function getCreateErrors(
   return errors;
 }
 
-export function getCreateTeamMemberError(user: User) {
+export function getCreateTeamMemberError(user: User, noEmail?: boolean | null) {
```

Comment:
Can we just rename and refactor this function? This is never used the originally intended way anymore, no need to make it flexible.

Reply:
Yeah definitely, I was just on the edge but decided for the quicker fix. 

I'll fix it up 👍

Reply:
Ah, now I'm really ashamed now that I see it has only one instance of use.